themes/default/layouts/page/cloud-engineering-summit.html

```diff
@@ -96,6 +96,39 @@ <h3>{{ $item.name }}</h3>
         </div>
     </section>
 
+    <section id="sponsors" class="container mx-auto mt-32">
+        <h2 class="text-center">Brought to you by</h2>
+        <div class="flex flex-wrap items-center justify-center">
+            <div class="w-full lg:w-1/5 flex items-center justify-center mx-8 my-4">
+                <img class="h-32" src="/logos/tech/aws-logo.svg" alt="AWS" />
+            </div>
+
+            <div class="w-full lg:w-1/5 flex items-center justify-center mx-8 my-4">
+                <img class="h-32" src="/logos/tech/bridgecrew-logo.svg" alt="Bridgecrew" />
+            </div>
+
+            <div class="w-full lg:w-1/5 flex items-center justify-center mx-8 my-4">
+                <img class="h-32" src="/logos/tech/ci-cd/codefresh.svg" alt="Codefresh" />
+            </div>
+
+            <div class="w-full lg:w-1/5 flex items-center justify-center mx-8 my-4">
+                <img class="h-32" src="/logos/tech/gcp-logo.svg" alt="Google Cloud" />
+            </div>
+
+            <div class="w-full lg:w-1/5 flex items-center justify-center mx-8 my-4">
+                <img class="h-32" src="/logos/tech/honeycomb-logo.svg" alt="Honeycomb" />
+            </div>
+
+            <div class="w-full lg:w-1/5 flex items-center justify-center mx-8 my-4">
+                <img class="h-32" src="/logos/tech/nginx.svg" alt="NGINX" />
+            </div>
+
+            <div class="w-full lg:w-1/5 flex items-center justify-center mx-8 my-4">
+                <img class="h-32" src="/logos/tech/ns1.png" alt="NS1" />
```

Comment:
not blocking, but this logo is getting significantly more weight than the others;if its easy to reduce its size a bit compared to the others and this wasn't intentional than id recommend fixing it